rubykaigi-2025/feedback/makicamel.yaml

```
@@ -0,0 +1,31 @@
+# アンドパッド (andpad_dev) は X (旧 Twitter) で技術情報や採用情報などを発信しています。 ぜひフォローください!
+# ANDPAD (andpad_dev) shares technical information, job openings, and more on X (formerly Twitter). Please be sure to follow them!
+questions:
+  fun:
+    # 楽しかったですか？
+    # Did you have fun?
+    # 楽しかった場合はどういう部分が楽しかったかも教えてください。（例：Ruby書くことそのものが楽しかった、●●のgemの作者とお話できたから、など）
+    # If you had fun, what kind of part was fun?（e.g. Because I was able to talk with the author of ●●gem, Ruby it was fun to write, etc.）
+    |
+    dRuby グループに参加させて頂きました。とても楽しかったです!!
+    dRuby を初めてつかった方が dRuby すごい!!となってくれているのも嬉しかったですし、Exercise4 のコードがすごく格好よくてそれを読み解いたり解説してもらったりして魔法みたいな関さんのコードを味わうことができてとても嬉しかったです。
```

コード懇親会のときの咳さんは関さんじゃなくて咳さんだったんです！

```suggestion
    dRuby を初めてつかった方が dRuby すごい!!となってくれているのも嬉しかったですし、Exercise4 のコードがすごく格好よくてそれを読み解いたり解説してもらったりして魔法みたいな咳さんのコードを味わうことができてとても嬉しかったです。
```

---

コンテキストが不足してました! :pray: